src/kasse.c

```diff
@@ -147,51 +147,16 @@ static signed int buy(char *name, unsigned int price) {
   BYTE c, x, y, nickname_len;
   int einheiten;
   char nickname[NICKNAME_MAX_LEN + 1];
-  char rest[11];
+  char rest[10];
```

Comment:
Given that rest is used for format_euro, I suggest we introduce a constant for 10 (9 characters for `%3d,%02dEUR` and a terminating NUL byte).

Reply:
ah, that happened because I blindly cargo-culted it.
Now I see where the size of the buffer comes from.
Will introduce a `#define`.